File: Integer/TwoSum/TwoSum.java
class TwoSum {
	public int[] twoSum(int[] nums, int target) {
		if(num == null || nums.length() == 0)
			return new int[] {0,0};
		}
		
		Map<Integer, Integer> result = new HashMap<Integer, Integer>();
		int index1 = 0, index2 = 0;
		for(int i = 0; i < nums.length(); ++i) {
			if(result.containsKey(target - nums[i])) {
				index1 = result.get(target - nums[i]);
				index2 = i;
				return new int[] {1 + index1, 1 +index2};
			} else {
				result.put(nums[i], 1);
			}
		}
		
		
		return new int[] {0, 0};
	}
}

File: Integer/TwoSum/TwoSum.py
class TwoSum:
    def twoSum(self, nums, target):
        hashDict = {}
        
        for i, item in enumerate(nums):
            if(target - item) in hashDict:
                return (hashDict[target - item] + 1, i + 1)
            hashDict[item] = i 
        
        
        return (-1, -1)